wrapanapi/systems/ec2.py

```diff
@@ -482,6 +483,66 @@ def cleanup(self):
         return self.delete()
 
 
+class Resource:
+    """
+    This class represents a resource returned by Resource Explorer.
+    """
+
+    def __init__(self, arn, region, resource_type, service, properties=[]):
+        self.arn = arn
+        self.region = region
+        self.resource_type = resource_type
+        self.service = service
+        self.properties = properties
+
+    def get_tag_value(self, key) -> str | None:
+        """
+        Returns a tag value for a given tag key.
+        Tags are taken from the resource properties.
+
+        Args:
+            key: a tag key
+        """
+        tags = self.get_tags(regex=f"^{key}$")
+        if len(tags) > 0:
+            return tags[0].get("Value")
+        return None
+
+    def get_tags(self, regex="") -> typing.List[dict]:
+        """
+        Returns a list of tags (a dict with keys 'Key' and 'Value').
+        Tags are taken from the resource properties.
+
+        Args:
+            regex: a regular expressions for keys, default is ""
+        """
+        list = []
+        if len(self.properties) > 0:
+            data = self.properties[0].get("Data")
```

Comment:
Any reason why we are not looping over all the properties and just the first property?

Does all properties have same tags in each of them ?

Reply:
So far, I found only the only one property. But it is possible they (AWS) will add some new propeties in the future, so I have changed that and loop through all properties.